x-pack/plugins/siem/public/components/page/hosts/events_table/index.tsx

```diff
@@ -230,4 +221,28 @@ const getEventsColumns = (
       </>
     ),
   },
+  {
+    name: i18n.MESSAGE,
+    sortable: false,
+    truncateText: true,
+    width: '25%',
+    render: ({ node }) => {
+      const message = getOr(null, 'message[0]', node);
+      const overflowLength = 50;
+      return message != null
+        ? getRowItemDraggable({
```

Comment:
it also seems weird to a search on a message in the timeline

Reply:
I feel like it will make more sense to break out the message to individual drag and drop like ip, host.name etc...

Reply:
I chatted a little with @andrew-goldstein about this. It does seem a little weird since a good portion of messages will have some unique value/identifier in them (PID, IP, etc), but it's nice for filtering on messages that are constant like `Generic Protocol Command Decode` and such.

I really like the idea of breaking out the message into individual DnD components. Let's explore this more as a future feature!